src/modelframe.jl

```diff
@@ -140,7 +140,9 @@ _droplevels!(x::Any) = x
 _droplevels!(x::AbstractCategoricalArray) = droplevels!(x)
 
 function ModelFrame(trms::Terms, d::AbstractDataFrame;
-                    contrasts::Dict = Dict())
+                    contrasts::Dict = Dict(), intercept::Bool = true)
+    trms = Terms((getfield(trms, fieldname) for fieldname in fieldnames(trms))...)
```

Comment:
I thought it was better to make a copy, in case some user wants to reuse these terms for a different model, but maybe there's no need for this line

Reply:
Makes sense, but isn't there a simpler way to make a copy of `Terms` objects, like `copy`?

Reply:
Surprisingly not... With `copy` I got:

```julia
julia> copy(StatsModels.Terms(@formula(x~y)))
ERROR: MethodError: no method matching copy(::StatsModels.Terms)
```

Reply:
OK. Probably worth defining it then. It's quite easy: just call the constructor and pass it all the fields (arrays do not need to be copied, that's `deepcopy`'s job).